source/options.html

```diff
@@ -12,20 +12,24 @@
 	form ~ form {
 		margin-top: 2em;
 	}
+	textarea {
+		width: 100%;
+	}
 </style>
 <form id="options-form">
 	<h2>Information</h2>
 	<p>
 		GitHub Enterprise user? While on your site, right-click on Refined GitHub's icon in the toolbar and select <strong>Enable Refined GitHub on this domain</strong>.
 		<a href="https://user-images.githubusercontent.com/1402241/32874388-e0c64150-cacc-11e7-9a50-eae3727fd3c2.png" target="_blank">Example</a>. This is <a href="https://github.com/sindresorhus/refined-github/issues/840#issuecomment-349936195" target="_blank">not yet supported by Firefox</a>.
 	</p>
-	
-	<h2>News feed</h2>
+
+	<h2>Disable features</h2>
 	<p>
-		<label>
-			<input type="checkbox" name="hideStarsOwnRepos">
-			Hide other users starring/forking your repos
-		</label>
+		List the features to disable, by <a href="https://github.com/sindresorhus/refined-github/tree/master/source/features" target="_blank">file name</a>: <br>
+		<textarea name="disabledFeatures" rows="5" placeholder="For example:
+log-active-features
+hide-own-stars
```

Comment:
It's weird to use literal newlines here. I would use `\n` escape instead.

Reply:
\n doesn’t work but this might https://stackoverflow.com/a/983988